runtime/autoload/remote/host.vim

```
@@ -200,6 +200,19 @@ function! s:RegistrationCommands(host) abort
     call remote#host#RegisterPlugin(host_id, path, [])
   endfor
   let channel = remote#host#Require(host_id)
+  " check the process started correctly
+  try
+    if rpcrequest(channel, 'poll') !=# 'ok'
+      " error should handled by on_exit on provider
+      return []
+    endif
+  catch
+    echoerr v:throwpoint
+    echoerr v:exception
+    throw a:host.' failed to start'
```

Can we just let the error bubble up, instead of catching and re-throwing?